prisma/schema.prisma

```diff
@@ -8,12 +8,26 @@ datasource db {
 }
 
 model ConnectInstallation {
-  id           Int    @id @default(autoincrement())
-  key          String
-  clientKey    String @unique
-  sharedSecret String
-  baseUrl      String
-  displayUrl   String
+  id               Int                @id @default(autoincrement())
+  key              String
+  clientKey        String             @unique
```

Comment:
Now that this is being used as an FK, I'm wondering if it would be better to make this the primary key of this table.

Reply:
Hold up, what are we actually using as the FK? I thought we would have only one table containing clientKey which other tables join on to get the value (meaning also that only one table needs to update that value and you can get those related tables to on delete cascade if desired). i.e.
```sql
CREATE TABLE connect_installation {
  id BIGSERIAL PRIMARY KEY,
  clientKey varchar,
  baseUrl varchar
  -- other columns
};

CREATE TABLE associated_design {
  connect_installation_id
    REFERENCES connect_installation ON DELETE CASCADE,
  fileKey varchar,
  nodeId varchar
  -- other columns
};

-- Select queries would look like
SELECT fileKey, nodeId
FROM associate_design
  JOIN connect_installation ON associate_design.connect_installation_id = connect_installation.id
WHERE connect_installation.clientKey = :clientKey;
```

Reply:
Good point, there's no need to use the `clientKey` as an FK here, can just use the `id` instead, I'll update.